output_image: "/yolo/image_raw"
source: 0 # file/dir/URL/glob/screen/0(webcam)
weights: "yolov5s.pt"  # model path or triton URL   
data: "/data/coco128.yaml"  # dataset.yaml path"
conf_thres: 0.25 # confidence threshold
iou_thres: 0.45 # NMS IOU threshold
max_det: 1000 # maximum detections per image
device: "" # cuda device, i.e. 0 or 0,1,2,3 or cpu
agnostic_nms: False # class-agnostic NMS
